js/main.js

```diff
@@ -394,11 +394,15 @@ function load_finish(loader_modal) {
     $('#fileupload').val('');
     return false;
   }
+
   vrvToolkit = new verovio.toolkit();
    if(mei.documentElement.namespaceURI != "http://www.music-encoding.org/ns/mei") {
      // We didn't get a MEI? Try if it's a musicXML
      try {
-       let new_svg = vrvToolkit.renderData(data, {pageWidth: 20000,
+       var xml_data = sanitize_xml(mei);
```

Comment:
Why? The "mei" here is something that isn't an MEI but rather something like a MusicXML or so, so it's not clear to me that it would make sense to screen for the same set of tags as for an MEI.

Reply:
But MEI is an XML document, no? The sanitization list is meant to include MEI and MusicXML tags, all of which are treated as XML tags.

Reply:
Just clarified this in the relevant `conf.js` comment.

Reply:
Sure - it might be something else entirely though, such as an ABC or Humdrum file, which just didn't get parsed as an XMLDocument, which is why I just feed the raw data to Verovio to see if it can figure something out.

I'm not sure if there is some overlap between the tag vocabularies of MusicXML and MEI that might mean that one would want to filter them in one and not the other, but I also wouldn't count on that _never_ happening.

Reply:
I see. I took `// We didn't get a MEI? Try if it's a musicXML` to mean that only MEI and MusicXML are to be supported by the app.

If we also support non-XML syntaxes, then I am going to remove the recently added `exit` statements a few lines above, just after `DOMParser.parseFromString()`, so that the app will not terminate, and will try to proceed with whatever input it was given.

I'll separate the MusicXML and MEI vocabularies shortly.

Reply:
Yeah, I should've updated that comment after Verovio got support for more formats. Sorry!

In particular, Verovio now also supports zipped MusicXML, so it might be the case that we _have_ a MusicXML file, but it hasn't been parsed into an `XMLDocument` even so.

Reply:
All well. I pushed a revision with separate vocabularies for old-school MusicXML and MEI. We can generalize that in future if we work with non-XML corpora.

Reply:
I'd prefer it to also restore this section (l 400-403) to how it was before, so that it uses the original raw data. Do you want me to do those changes?

Reply:
Didn't I do that already? I'm out at the moment, but will do it later otherwise.

Reply:
Nope, not as far as I can tell - I'll sort it out.